QueryBuilder/Parsing/Filter.php

```diff
@@ -137,7 +137,11 @@ public function compile()
         if (in_array($this->parsedInput['comparator'], array(self::COMPARATOR_IS_NULL, self::COMPARATOR_IS_NOT_NULL))) {
             $value = $this->parsedInput['comparator'];
         } else {
-            $value = $this->parsedInput['comparator'] . ':' . $this->parsedInput['value'];
+            $parsedValue = $this->parsedInput['value'];
+            if (is_array($this->parsedInput['value'])) {
+                $parsedValue = implode(',', $this->parsedInput['value']);
```

Comment:
Thanks for the fix, seems to be OK. One thing I would like to ask you - please introduce a constant, i.e ARRAY_DELIMITER to this class which would contain ',' value and then update it here in implode() function as well as in https://github.com/sergeil/SliExtJsIntegrationBundle/blob/master/QueryBuilder/Parsing/Filter.php#L99.

Reply:
Also, adding a test before the doing the fix would be helpful :)